README.md

```diff
@@ -1,2 +1,54 @@
 # community
-Community repository for governance, issue with org, scope, meetings etc.
+
+# what is the just-foss organization?
+
+The aim of just-foss organisation it to be a free, opensource, remote, regular meeting, written by and for hackers in computer science. 
```

Comment:
To which one do we stick? Organization vs organisation?

Reply:
organization probably? what do you prefer